packages/firestore/test/integration/api/database.test.ts

```diff
@@ -1373,6 +1373,25 @@ apiDescribe('Database', (persistence: boolean) => {
       }
     };
 
+    class ModelWithRef {
```

Comment:
Could you re-use the Post-Converter? That might cut down on the extra lines in this file.